python/lsst/ts/m2gui/controltab/tab_settings.py

```diff
@@ -50,14 +50,27 @@ class TabSettings(TabDefault):
     LOG_LEVEL_MINIMUM = 10
     LOG_LEVEL_MAXIMUM = 50
 
+    # The unit is seconds
     TIMEOUT_MINIMUM = 1
 
+    # The unit is Hz
+    REFRESH_FREQUENCY_MINIMUM = 1
+    REFRESH_FREQUENCY_MAXIMUM = 10
+
+    POINT_SIZE_MINIMUM = 9
+    POINT_SIZE_MAXIMUM = 14
+
     def __init__(self, title, model):
         super().__init__(title, model)
 
         self._settings = self._create_settings()
 
-        self._button_apply = set_button("Apply Settings", self._callback_apply)
+        self._button_apply_host = set_button(
```

Comment:
This smell like a lousy UI design. What is the host and what are the general settings? What will be updated when the button is pushed? Consider using e.g. tabs (QTabWidget), where the "apply settings" button will trigger an update only of settings in the given tab - that will provide the operator a clear split of what is the host and what is considered general. Or some other approach, like putting general/host settings into QGroupBox.

Reply:
The general/host settings are under the different `QGroupBox` already. Thanks!